eslint-common.json

```diff
@@ -9,6 +9,7 @@
   "dot-location": ["property"],
   "eol-last": ["always"],
   "eqeqeq": ["always", {"null": "never"}],
+  "for-direction": [],
```

Comment:
Ah, I see. :)

I'm [using](https://github.com/wearereasonablepeople/eslint-config-warp/blob/master/es6.js#L5) eslint's built-in extension capability to share rules between configurations in the config I maintain, as opposed to a build process. Any reason to favour the build option?